### PR TITLE
[tw] Localize Site Reliabilitgy Engineering for Traditional Chinese (#2372)

### LINKED ISSUE #2372: [tw] Localize `Site Reliability Engineering` for Traditional Chinese
1. Localize `Site Reliability Engineering` for Traditional Chinese
2. https://glossary.cncf.io/site-reliability-engineering/

### PR DESCRIPTION
**Pull request by Fionn88**
### Describe your changes

Origin content from https://glossary.cncf.io/site-reliability-engineering/

### Related issue number or link (ex: `resolves #issue-number`)

resolves #2372 

### Checklist before opening this PR (put `x` in the checkboxes)
- [x] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [x] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

**Comment by ydFu** [comment]
The commit did not pass the DCO check because it was not signed during the git commit process. Please use "git commit -s -m" for submission.

File: content/zh-tw/site-reliability-engineering.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+確保應用程式[可靠](/zh-tw/reliability/)運行需要多種能力，
+從性能監控、警報、除錯到故障排除都是必要的。
+如果缺少這些能力，系統操作員只能對問題做出反應，
+而無法主動努力避免它們 - 停機只是時間問題。
+
+
+## 如何幫助我們 {#how-it-helps}
+
+網站可靠性工程透過持續改進底層系統，
+以降低軟體開發過程的成本、時間和工作量。
```
Comment (ydFu) [inline content/zh-tw/site-reliability-engineering.md:29]:
```suggestion
以最大限度地降低軟體開發過程的成本、時間和工作量。
```

File: content/zh-tw/site-reliability-engineering.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+該系統不斷測量和監控基礎設施和應用元件。
```
Comment (ydFu) [inline content/zh-tw/site-reliability-engineering.md:30]:
```suggestion
該系統持續測量和監控基礎設施和應用程式的元件。
```

File: content/zh-tw/site-reliability-engineering.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+當出現問題時，系統會指導網站可靠性工程師何時、在哪裡以及如何修復它。
```
Comment (ydFu) [inline content/zh-tw/site-reliability-engineering.md:31]:
```suggestion
當出現問題時，系統會向網站可靠性工程師指示何時、何處以及如何修復它。
```

File: content/zh-tw/site-reliability-engineering.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+這種方法通過自動化維運任務有助於創建高度[可擴展](/zh-tw/scalability/)和可靠的軟體系統。
```
Comment (ydFu) [inline content/zh-tw/site-reliability-engineering.md:32]:
```suggestion
這種方法通過自動化操作的任務，有助於創建高度的[擴展性](/zh-tw/scalability/)和可靠性的軟體系統。
```
en:
```
This approach helps create highly [scalable](/scalability/) and reliable software systems by automating operational tasks.
```

**Review by ydFu — COMMENTED** [review]
Thank for your contribution,
Try to keep the line breaks the same as the original text.
[Please use semantic line breaks (one sentence per line) to ease review.](https://glossary.cncf.io/style-guide/#three-subheadings)

File: content/zh-tw/site-reliability-engineering.md [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+---
+
+## 是什麼 {#what-it-is}
+
+網站可靠性工程(SRE)是結合了維運和軟體工程。
+它主要應用於基礎架構和維運問題。
+網站可靠性工程師不是建構產品功能，
+而是建構系統來運行應用程式。
+雖然SRE和[DevOps](/zh-tw/devops/)有一些相似之處，
+但是DevOps主要關注將程式碼部署到生產環境，
```
[the remark this answers is not in the document]
Comment (Fionn88) [reply content/zh-tw/site-reliability-engineering.md:14]: [tw] fix one sentence per line

**Comment by Fionn88** [comment]
> Use rebase to squash/fixup dummy/unnecessary commits into only one commit.

Okay, no problem.

**Pull request closed without merging**